### ISSUE TITLE
Feature Request: Cursor hiding

### ISSUE THREAD

**Issue by teraspora** [patriciogonzalezvivo/glslViewer#122]
It would be good to be able to hide the mouse cursor while retaining its functionality.

**Comment by patriciogonzalezvivo** [comment]
You can hide/show the cursor in run time using the command "cursor,on" or  "cursor,off" beside it's or not showing the uniform "u_mouse" is always enabled

**Comment by teraspora** [comment]
This doesn't work for me.
I'm running GlslViewer 1.5.6 on Ubuntu 18.04.1 LTS.
The command sets the cursor to OFF but does not hide it:
// > cursor, off
// > cursor
off

This SO answer may shed some light:  https://stackoverflow.com/questions/4431637/hiding-mouse-cursor-with-glfw

"
Since glfw 3.0 the API call has changed, you must use glfwSetInputMode with a pointer of your window.

glfwSetInputMode(window, GLFW_CURSOR, GLFW_CURSOR_HIDDEN);

For more information see http://www.glfw.org/docs/3.0/moving.html
"

**Comment by patriciogonzalezvivo** [comment]
is there a space?

**Comment by patriciogonzalezvivo** [comment]
I will take a look

**Comment by teraspora** [comment]
In fact cursor on/off seems to be off by default.   Setting `cursor on` shows a white cross cursor underneath the system cursor.
So this white cross cursor responds to the on/off commands.   But the system cursor - in my case an upward pointing arrow - remains unaffected by the commands.

**Comment by patriciogonzalezvivo** [comment]
You are right. Sorry for that. I could get remove the arrow cursor completely and add the cross cursor ON by default. That would make sense? do you think is what most of the users will expect?
